### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Review by bencharp — COMMENTED** [review]
I don't really master this rewriting thing but when I try to apply the recommendations of yunohost it crashes... 

I just followed the README of 
https://github.com/remotestorage/armadietto#serving-over-https

**Comment by ericgaspar** [comment]
Nothing as changed regarding https

```
    if ($scheme = http) {
        rewrite ^ https://$server_name$request_uri? permanent;
    }

```

This part as been removed from the NGINX because it is, since 4.3, part of NGINX core.

**Comment by bencharp** [comment]
> This part as been removed from the NGINX because it is, since 4.3, part of NGINX core.

Ok :slightly_smiling_face: 
I think I misinterpreted the linter's comment about `rewrite`
I still have a little trouble understanding this redirection stuff... :confused:

**Comment by ericgaspar** [comment]
!testme

File: manifest.json [hunk 1]
```diff
@@ -3,8 +3,8 @@
     "id": "armadietto",
     "packaging_format": 1,
     "description": {
-        "en": "A remoteStorage server written for Node.js.",
-        "fr": "Un serveur remoteStorage écrit pour node.js."
+        "en": "RemoteStorage server written for node.js",
```
Comment (bencharp) [inline manifest.json]: remoteStorage.js is only one client library for using/integrating the remoteStorage protocol.
It always writes in camelCase and is considered as a remoteStorage server running as a nodejs web service
cf : recent comment in this PR https://github.com/remotestorage/armadietto/pull/37

**Comment by bencharp** [comment]
For the description, if you're ok, I would suggest this:

```shell
"description" : {
        "en": "A remoteStorage server running as a nodejs web service with systemd",
        "fr": "Un serveur remoteStorage excécuté en tant que service web nodejs par systemd"
}
```
Thanks for all fixes :+1:

**Review by bencharp — APPROVED** [review]

**Pull request merged**